pages/common/r2e.md

```diff
@@ -0,0 +1,24 @@
+# r2e
+
+> Forwards RSS feeds to your email address. More information: https://github.com/rss2email/rss2email.
+> Requires a configured sendmail or smtp setup.
+
+- Create a new feed database that sends email to your email address:
```

Comment:
Generally we try and about `you` / `your` / etc in tldr pages. 
Perhaps this will read better?
```suggestion
- Create a new feed database that sends email to an email address:
```

Reply:
For some weird reason my tldr client renders multiple lines as one, so I thought that was the normal format.